.github/github_org_control/configs.py

```diff
@@ -14,8 +14,8 @@
 from pathlib import Path
 
 
-if sys.hexversion < 0x3060000:
-    raise Exception("Python version must be >= 3.6")
+if sys.version_info.minor < 7:
```

Comment:
IMO when we check version of the package, we need to check major and minor number of version

Reply:
> IMO when we check version of the package, we need to check major and minor number of version

Consider check version:
```python
sys.version_info[:3] < (3, 6, 0)
```

Reply:
Thank you for comments, I applied @praasz suggestion, really cool way of checking Python version. :)